helm/hpcc/values.yaml

```diff
@@ -385,7 +385,7 @@ dafilesrv:
   application: spray
   service:
     servicePort: 7300
-    visibility: local
+    visibility: cluster
```

Comment:
NB: this should always have been 'cluster'. We do not want the spray service (which serves the local dfuserver) to have any external access.